diff --git a/core/DTNHost.java b/core/DTNHost.java
index 6a8c6c9..e5df8f1 100644
--- a/core/DTNHost.java
+++ b/core/DTNHost.java
@@ -17,7 +17,7 @@ import routing.util.RoutingInfo;
  * A DTN capable host.
  */
 public class DTNHost implements Comparable<DTNHost> {
-	private static int nextAddress = 0;
+	private static int nextAddress = 1;
 	private int address;
 
 	private Coord location; 	// where is the host
@@ -102,7 +102,7 @@ public class DTNHost implements Comparable<DTNHost> {
 	 * Reset the host and its interfaces
 	 */
 	public static void reset() {
-		nextAddress = 0;
+		nextAddress = 1;
 	}
 
 	/**
diff --git a/input/ExternalEventsQueue.java b/input/ExternalEventsQueue.java
index 108540a..c99294d 100644
--- a/input/ExternalEventsQueue.java
+++ b/input/ExternalEventsQueue.java
@@ -29,7 +29,7 @@ public class ExternalEventsQueue implements EventQueue {
 	private ExternalEventsReader reader;
 	private int nextEventIndex;
 	private int nrofPreload;
-	private List<ExternalEvent> queue;
+	protected List<ExternalEvent> queue;
 	private boolean allEventsRead = false;
 	
 	/**
diff --git a/one.sh b/one.sh
index 865616e..250dffc 100755
--- a/one.sh
+++ b/one.sh
@@ -1,2 +1,2 @@
 #! /bin/sh
-java -Xmx512M -cp .:lib/ECLA.jar:lib/DTNConsoleConnection.jar core.DTNSim $*
+java -Xmx512M -cp .:bin:lib/ECLA.jar:lib/DTNConsoleConnection.jar:$CGR_JNI_CLASSPATH core.DTNSim $*
diff --git a/report/MessageStatsReport.java b/report/MessageStatsReport.java
index 33fe2d8..d5f2d6d 100644
--- a/report/MessageStatsReport.java
+++ b/report/MessageStatsReport.java
@@ -176,7 +176,11 @@ public class MessageStatsReport extends Report implements MessageListener {
 			;
 		
 		write(statsText);
+		write(getMoreInfo());
 		super.done();
 	}
 	
+	protected String getMoreInfo() {
+		return "";
+	}
 }
